pkgs/development/r-modules/default.nix

```diff
@@ -938,6 +938,21 @@ let
   ];
 
   otherOverrides = old: new: {
+    gifski = old.gifski.overrideDerivation( attrs: rec {
```

Comment:
I am not too sure about the order of these overrides. It does not appear to be alphabetical, so I temporarily placed it at the top

Reply:
This is fine, it's unordered these days.